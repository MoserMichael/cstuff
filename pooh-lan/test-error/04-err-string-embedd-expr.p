

a ='string with wrong express [ %%% ] what now ?'
