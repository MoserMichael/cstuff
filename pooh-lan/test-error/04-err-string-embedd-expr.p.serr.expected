
test-error/04-err-string-embedd-expr.p(3): Error: the function undef is not defined, can't call a function that is not defined
      3 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |a = 'multiline string with wrong express [ undef() ] what now [ 2 * 'aaaa' ]
        |...........................................^^^^^

test-error/04-err-string-embedd-expr.p(4): Error: the function undef is not defined, can't call a function that is not defined
      4 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        | another embeaded expr [ undef() ] that call undefined function. what now ? [ 'bbbbb' * 3 ] ? '
        |.........................^^^^^
Type checking failed
